packages/wasm/src/services/blockchain/service.ts

```diff
@@ -69,6 +96,19 @@ export class BlockchainService {
     this.resolver = this.createDIDResolver();
   }
 
+  getTypesForDIDOrAccumulator(did) {
+    const didType = did.split(':')[1];
+    const chainDidType =
+      !this.dock && didType === 'dock' ? 'cheqd' : didType;
```

Comment:
this doesnt account for legacy accumulator IDs of dock:accumulator:

also, this isnt used for DIDs only so the variable naming is a bit off putting

Reply:
Got it. I've just copied the logic from certs monorepo https://github.com/docknetwork/web/blob/f9cf9a4da49949f3c78fe0a0da32eb432c2ded8b/apps/api/src/strategies/registries/AccumulatorRevocationRegistryStrategy.js#L73. 

I can do some refactoring on that. I'm not pretty sure on the fix for  legacy accumulator IDs of dock:accumulator:.

Should we apply that on certs as well? or do you have any other reference for the ideal implementation?